dodola/services.py

```diff
@@ -718,6 +721,11 @@ def clean_cmip6(x, out, leapday_removal):
         Whether or not to remove leap days.
     """
     ds = storage.read(x)
+
+    # Cannot have chunks in time dimension for 360 day calendar conversion so loading
+    # data into memory.
+    ds.load()
```

Comment:
@brews I had thought of doing this directly in `xclim_convert_360_days*`. Why do you want to do this more upstream ?

Note that a rechunking is happening at the end of `standardize_gcm`.

Reply:
@emileten Yeah, this is a bit hairy. Talking this out:

I think Gergel originally started doing it with QDM and QPLAD in order to get things running for her. I think I'd rather the `.load()` be here in dodola.services rather than dodola.core because there is a chance that we might use the dodola.core functions in a more advanced script and workaround -- I feel like it would be a pain to have that "load()" embedded there if you had your data correctly chunked and you were trying to stream a crap load of data.

Also, as you might smell from the above example, the `.load()` isn't really core to why people would call this `dodola.core.standardize_gcm` function...it's more of a storage/streaming implementation detail — which we're trying to keep out of our core logic. It isn't part of the "business logic".

But with the chunk() that we DO have in `dodola.core`... it's an explicit part of standardizing the GCM data. When we ask one of the climate scientists what the cleaning and standardizing steps involves, they will mention the need to have common chunking. So, this hard rechunking is allowed because its something very deliberate in the purpose of this standardizing function and not just a side effect of our implementation that we want to hide away.

Reply:
@brews go it. Thanks 👍